great-idea/css/index.css

```diff
@@ -62,3 +62,100 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 15px;
 }
+
```

Comment:
Nice work getting very precise with the CSS box model layout! I know it was probably frustrating, but hopefully it gave you some good insight into the power of flexbox.